src/btree/bt_split.c

```diff
@@ -1421,7 +1421,8 @@ __split_multi_inmem(WT_SESSION_IMPL *session, WT_PAGE *orig, WT_MULTI *multi, WT
             continue;
 
         if (supd->ins == NULL) {
-            slot = WT_ROW_SLOT(orig, supd->ripcip);
+            /* Note: supd->ins is never null for column-store. */
+            slot = WT_ROW_SLOT(orig, supd->rip);
```

Comment:
I'd suggest asserting this in the code.

Reply:
All the places? (I think I commented them all) Or is it sufficient just to put it where where supd is constructed?

Reply:
(conclusion from quick discussion: at the top is sufficient. I've done that)